bin/powershell/perfhelper.ps1

```diff
@@ -0,0 +1,81 @@
+##################################################

```

Comment:
Lets include the standard header (you can still have this below): https://github.com/sensu-plugins/documentation/blob/master/docs/developer_guidelines.md#coding-style you can tweak it to be less ruby specific of course.

Reply:
Sure, no problem. I will have at look at it tomorrow